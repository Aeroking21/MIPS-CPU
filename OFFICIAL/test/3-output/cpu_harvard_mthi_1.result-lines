RESULT =  00000123
