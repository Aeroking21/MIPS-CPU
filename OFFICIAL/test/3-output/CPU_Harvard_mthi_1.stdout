WARNING: test/ROM.v:21: $readmemh(test/testcases/mthi_1_instructions.mem): Not enough words in the file for the requested range [-1077936128:-1077933128].
Instruction = 24
Instruction r = 24
WARNING: rtl/RAM.v:53: $readmemh(data.mem): Not enough words in the file for the requested range [256:1000].
VCD info: dumpfile CPU_TB.vcd opened for output.
ROM MODULE RESULTS:
CPU started
REG_V0 =  00000000
REG_V0 =  00000000
REG_V0 =  00000000
REG_V0 =  00000000
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
REG_V0 =  00000123
RESULT =  00000123
